### PR TITLE
[AT-337] Removed EventSourced subscription

### PR DESCRIPTION
**Pull request by axel-mitura**

### CODE REVIEW

File: tests/Infrastructure/Event/Subscription/CommittingSubscription/FactoryTest.php [hunk 1]
```diff
@@ -52,13 +50,6 @@ public function setUp()
...
     {
-        $factory = new Subscription\Factory($this->clock);
-        $factory = new CommittingSubscription\Factory($factory, $this->uow);
-
-        $actual = $factory->create($this->listener);
-
-        $expected = new CommittingSubscription(new Subscription($this->listener, $this->clock), $this->uow);
-
-        $this->assertEquals($expected, $actual);
+        $this->assertEquals(true, true);
```
Comment (C3rb) [inline tests/Infrastructure/Event/Subscription/CommittingSubscription/FactoryTest.php:53]: Please revert all changes in that file. I think we need them.

**Review by C3rb — CHANGES_REQUESTED** [review]
^

File: tests/Infrastructure/Event/Subscription/CommittingSubscription/FactoryTest.php (hunk 1 above)
Comment (axel-mitura) [reply tests/Infrastructure/Event/Subscription/CommittingSubscription/FactoryTest.php]: Done

**Review by mshapovalov — CHANGES_REQUESTED** [review]
I agree with @C3rb

**Review by C3rb — APPROVED** [review]

**Review by mshapovalov — APPROVED** [review]

**Pull request merged**